src/webgpu/shader/validation/expression/call/builtin/acosh.spec.ts

```diff
@@ -33,15 +34,15 @@ Validates that constant evaluation and override evaluation of ${builtin}() rejec
       .combine('stage', kConstantAndOverrideStages)
       .combine('type', kAllFloatScalarsAndVectors)
       .filter(u => stageSupportsType(u.stage, u.type))
-      .combine('value', kMinusOneToTwo)
+      .combine('value', linearRange(-2000, 2000, 10))
```

Comment:
This range is so wide that it misses useful granularity at 0, 1, 2.

Please include kMinusTwoToTwo

actually, I suggest using`  u => unique(kMinusTwoToTwo, fullFPRangeForType(u.type)) `

Reply:
Done